vision/object_detection_segmentation/yolov4/README.md

```diff
@@ -0,0 +1,93 @@
+# YOLOv4
+
+## Description
+[YOLOv4](https://github.com/hunglc007/tensorflow-yolov4-tflite) optimizes the speed and accuracy of object detection. It is twice faster than EfficientDet and improves YOLOv3's AP and FPS by 10% and 12%, respectively. 
+
+## Model
+
+|Model        |Download  |Download (with sample test data)|ONNX version|Opset version|Accuracy |
+|-------------|:--------------|:--------------|:--------------|:--------------|:--------------|
+|YOLOv4       |[251 MB](model/yolov4.onnx) |[236 MB](model/yolov4.tar.gz)|1.6 |11 |mAP of 0.5733 |
+
+
+
+## Inference
+### Conversion
+A tutorial for the conversion process can be found in the [conversion](dependencies/conversion.ipynb) notebook.
+
+Validation of the converted model and a graph representation of it can be found in the [validation](dependencies/onnx-model-validation.ipynb) notebook.
+### Running inference
+A tutorial for running inference using onnxruntime can be found in the [ort](dependencies/ort.ipynb) notebook.
+### Input to model
+Input images are resized to the shape `(1, 416, 416, 3)`.
```

Comment:
Describe the input dimensions. Likely something like -> (Batch-size, height, width, channels) but confirm.